Marlin/src/module/stepper_indirection.cpp

```diff
@@ -545,7 +547,7 @@
   void tmc_init(TMCMarlin<TMC2660Stepper, AXIS_LETTER, DRIVER_ID> &st, const uint16_t mA, const uint16_t microsteps, const uint32_t, const float, const bool) {
     st.begin();
 
-    static constexpr int8_t timings[] = CHOPPER_TIMING; // Default 4, -2, 1
+    static constexpr int timings[] = CHOPPER_TIMING; // Default 4, -2, 1
```

Comment:
Seems all these `timings` declarations can be dropped if you use `chopper_settings` instead, since it contains the same values.

Reply:
It sure looks that way yes. I'll do it tomorrow if you don't get to it first.